pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -71,19 +73,30 @@ type Applier interface {
 	ApplyRuntimeObject(obj runtime.Object, scheme *runtime.Scheme) (hiveresource.ApplyResult, error)
 }
 
-// Add creates a new SyncSet controller and adds it to the manager with default RBAC. The manager will set fields on the
+// Add creates a new SyncSetInstance controller and adds it to the manager with default RBAC. The manager will set fields on the
 // controller and start it when the manager starts.
 func Add(mgr manager.Manager) error {
-	return AddToManager(mgr, NewReconciler(mgr))
+	logger := log.WithField("controller", controllerName)
+	reapplyInterval := defaultReapplyInterval
+	var err error
+	if envReapplyInterval := os.Getenv(reapplyIntervalEnvKey); len(envReapplyInterval) > 0 {
+		reapplyInterval, err = time.ParseDuration(envReapplyInterval)
```

Comment:
You do not want to change `reapplyInterval` when there is an error parsing the environment variable.

Reply:
I should have been returning the error here. In that case it would be okay to change `reapplyInterval` right?

Reply:
Yes.

Reply:
While in that code, can you also bring `var err error` done into the scope that it will be used.